git-deblog

```diff
@@ -49,8 +59,11 @@ def main():
 
 
 def run(*args, **kwargs):
+    from locale import getpreferredencoding
     from subprocess import check_output
-    return check_output(*args, **kwargs).strip().decode(ENC)
+    encoding = getpreferredencoding()
```

Comment:
Yeah, is a tricky issue always with encodings... I'm trusting Python and the environment to set the correct encodings, but I have the feeling we'll have this problem also with `git config` stuff (an author name in UTF-8 and a poorly configured environment). I'll add options to override this via command-line option in the (very near) future.